Formula/vis.rb

```diff
@@ -35,24 +38,35 @@ def install
       system "luarocks", "build", "lpeg", "--tree=#{luapath}"
     end
 
-    system "./configure", "--prefix=#{prefix}"
+    system "./configure", "--prefix=#{prefix}", "--enable-lua"
```

Comment:
Added flag to make build fail on lua rather than silently ignoring if unable to find:
```
checking for liblua >= 5.2 ...
 checking for lua... no
 checking for lua5.3... no
 checking for lua5.2... no
 checking for lua-5.3... no
 checking for lua-5.2... no
 checking for lua53... no
 checking for lua52... no
./configure: cannot find liblua
```

May want to manually try test: https://github.com/martanne/vis/blob/v0.7/configure#L429-L453

The unversioned `lua` works on macOS.

Could also patch in versioned `lua5.4` (https://github.com/martanne/vis/commit/f02fd320174179f7d086cc93eb625d11bf59fb5a), but I am not sure if it will help when unversioned test failed.